src/main/kotlin/uk/gov/justice/digital/hmpps/externalusersapi/resource/UserController.kt

```diff
@@ -82,6 +87,64 @@ class UserController(private val userSearchService: UserSearchService) {
       status
     )
 
+  @PutMapping("/{userId}/enable")
+  @PreAuthorize("hasAnyRole('ROLE_MAINTAIN_OAUTH_USERS', 'ROLE_AUTH_GROUP_MANAGER')")
+  @ResponseStatus(HttpStatus.OK)
+  @Operation(
+    summary = "Enable a user.",
+    description = "Enable a user."
+  )
+  @ApiResponses(
+    value = [
+      ApiResponse(
+        responseCode = "200",
+        description = "OK.",
+        content = [
+          Content(
+            mediaType = "application/json",
+            schema = Schema(implementation = EmailNotificationDto::class)
+          )
+        ]
+      ),
+      ApiResponse(
+        responseCode = "401",
+        description = "Unauthorized.",
+        content = [
+          Content(
+            mediaType = "application/json",
+            schema = Schema(implementation = ErrorResponse::class)
+          )
+        ]
+      ),
+      ApiResponse(
+        responseCode = "403",
+        description = "Unable to enable user, the user is not within one of your groups.",
+        content = [
+          Content(
+            mediaType = "application/json",
+            schema = Schema(implementation = ErrorResponse::class)
+          )
+        ]
+      ),
+      ApiResponse(
+        responseCode = "404",
+        description = "User not found.",
+        content = [
+          Content(
+            mediaType = "application/json",
+            schema = Schema(implementation = ErrorResponse::class)
+          )
+        ]
+      )
+    ]
+  )
+  suspend fun enableUserByUserId(
+    @Parameter(description = "The userId of the user.", required = true) @PathVariable
+    userId: String
+  ) = userService.enableUserByUserId(
```

Comment:
I would probably swap this to userId: UUID - so that validation takes care of itself (and can then change Service call to take a UUID)

Reply:
changed to UUID